test/integration/external-accounts.test.js

```diff
@@ -220,4 +227,63 @@ describe("External Accounts", function () {
         });
     });
   });
+
+  describe("PATCH /external-accounts/discord-sync", function () {
+    let jwtToken, fetchStub;
+
+    beforeEach(async function () {
+      const userId = await addUser(userData[4]);
+      jwtToken = authService.generateAuthToken({ userId });
```

Comment:
Change token name to some variable name specific to SUPERUSER

Reply:
done